src/equiv.js

```diff
@@ -12,8 +12,11 @@ QUnit.equiv = (function() {
 	function useStrictEquality( b, a ) {
 
 		/*jshint eqeqeq:false */
-		if ( b instanceof a.constructor || a instanceof b.constructor ) {
+		if ( b instanceof a.constructor && a instanceof b.constructor ) {
 
+			// Compare same constructor.
+			return a.toString() == b.toString();
```

Comment:
@gibson042

Thank you. :)
I have misunderstood it. but I get it now. Check constuctor and check typeof before I understand your comment.

Good comments. I updating it soon.